NHapi20/NHapi.Base/Model/Primitive/TSComponentOne.cs

```diff
@@ -279,7 +279,7 @@ public virtual DateTime GetAsDate()
 		{
 			try
 			{
-				string[] dateFormats = new string[] {LongDateTimeFormat, ShortDateTimeFormat, LongDateTimeFormatWithSecond, LongDateTimeFormatWithOffset };
+				string[] dateFormats = new string[] {LongDateTimeFormat, ShortDateTimeFormat, LongDateTimeFormatWithSecond, LongDateTimeFormatWithOffset, LongDateTimeFormatWithFactionOfSecond };

```

Comment:
This file should not be included if the purpose is just to remove the .vs folder. Remove this file and and create a separate PR for this file.